requirements.txt

```diff
@@ -2,7 +2,6 @@
 # These requirements are only needed for dev/testing, the user requirements
 # are in setup.py
 #
-mock==1.0.1
-httpretty==0.8.3
-pylint==1.1.0
-nose==1.3.0
+future
```

Comment:
I prefer requirements with `==`

If in the future any of these libraries push a change which is not back-compatible our project might be affected.

Reply:
Okay, no problem. I will freeze it.